src/sage/parallel/map_reduce.py

```diff
@@ -1136,21 +1136,12 @@ def start_workers(self):
             sage: from sage.parallel.map_reduce import RESetMapReduce
             sage: def children(x):
             ....:     print(f"Starting: {x}", flush=True)
-            ....:     sleep(float(0.5))
-            ....:     print(f"Finished: {x}", flush=True)
             ....:     return []
             sage: S = RESetMapReduce(roots=[1, 2], children=children)
             sage: S.setup_workers(2)
-            sage: S.start_workers(); sleep(float(0.4))
+            sage: S.start_workers(); sleep(float(5))  # long time
```

Comment:
The `# long time` label causes a test failure in the cleanup section, when running tests in non-long mode.

Reply:
https://github.com/sagemath/sage/pull/38873